### PR TITLE
LPS-116130 Add columns to search container.

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci.test:relevant

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/ContentDashboardItem.java [hunk 1]
```diff
@@ -25,6 +28,9 @@
  */
 public interface ContentDashboardItem<T> {
 
+	public List<AssetCategory> getAssetCategories(
```
Comment (darquesdev) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/ContentDashboardItem.java:31]: Why don't you use just vocabularyId?

**Pull request closed without merging**